src/warnet/graph.py

```diff
@@ -98,119 +98,90 @@ def custom_graph(
 
 def inquirer_create_network(project_path: Path):
     # Custom network configuration
-    questions = [
-        inquirer.Text(
-            "network_name",
-            message=click.style("Enter your network name", fg="blue", bold=True),
-            validate=lambda _, x: len(x) > 0,
-        ),
-        inquirer.List(
-            "nodes",
-            message=click.style("How many nodes would you like?", fg="blue", bold=True),
-            choices=["8", "12", "20", "50", "other"],
-            default="12",
-        ),
-        inquirer.List(
-            "connections",
-            message=click.style(
-                "How many connections would you like each node to have?",
-                fg="blue",
-                bold=True,
-            ),
-            choices=["0", "1", "2", "8", "12", "other"],
-            default="8",
+    network_name = inquirer.text(
+        message=click.style("Enter your network name", fg="blue", bold=True),
+        validate=lambda _, x: len(x) > 0,
+    )
+    nodes = inquirer.list_input(
+        message=click.style("How many nodes would you like?", fg="blue", bold=True),
+        choices=["8", "12", "20", "50", "other"],
+        default="12",
+    )
+    if nodes == "other":
+        nodes = inquirer.text(
+            message=click.style("Enter the number of nodes:", fg="blue", bold=True),
```

Comment:
>Enter the number of nodes:

Leave off the ":" because inquirer will add one automatically.

Reply:
fixed in 711156a73898eda58a2e306fcc21a58356966d95